Frontend/src/app/layouts/admin-layout/pages/not-selected-students/not-selected-students.component.ts

```diff
@@ -1,15 +1,20 @@
 import { Component, OnInit } from '@angular/core';
-
+import { Student } from '../../models/student.model';
+import { StudentService } from '../../../../Services/student.service';
 @Component({
   selector: 'app-not-selected-students',
   templateUrl: './not-selected-students.component.html',
   styleUrls: ['./not-selected-students.component.scss']
 })
 export class NotSelectedStudentsComponent implements OnInit {
-
-  constructor() { }
+  students : Student[];
+  batchId : String = "batch16";
```

Comment:
I think we need an API to get the latest batch or ?

Reply:
I thought to load batches to a dropdown. and  from there admin selects the batch she wants.

Reply:
Ahhh thats fine. But what will be the initial value before selecting the dropdown?

Reply:
It should be current batch